### ISSUE TITLE
Feature: speech recognition for visual feedback on audio

### ISSUE THREAD

**Issue by robinp** [owickstrom/komposition#94]
Could wire up speech recognition on the audio chunks to:
- auto-name the clips when importing
- show the text an the audio block in the timeline

Bonus: add keyword / topic extraction.

**Comment by owickstrom** [comment]
Yes, this is something I've been wanting to add. But I haven't found any good tools to integration with yet (maybe haven't looked hard enough...)

**Comment by robinp** [comment]
You mean speech recognition? I had good experience with [CMU PocketSphinx](https://github.com/cmusphinx/pocketsphinx), even wrote some c2hs bindings. If you are interested, I can dig them up!

**Comment by owickstrom** [comment]
Sorry for dropping the ball on this one. PocketSphinx seems interesting, and it's available on macOS, Linux, and Windows, it seems. Do you have the bindings published somewhere?

**Comment by robinp** [comment]
Just pushed after some dusting: https://github.com/TreeTide/voicetrans/tree/master/sphinx . Partial bindings, but good enough to run the recognition (see `app/Main.hs` to get model file and test input).
